src/lda.jl

```diff
@@ -284,7 +284,7 @@ Transform input sample(s) in `x` to the output space of MC-LDA model `M`. Here,
 predict(M::MulticlassLDA, x::AbstractVecOrMat{T}) where {T<:Real} = M.proj'x
 
 """
-    fit(MulticlassLDA, X, y; ...)
+    fit(Type{MulticlassLDA}, X, y; ...)
```

Comment:
Please, revert this. In documentation, an example of function call is provided. The first parameter has to be a type and not a type representation object.
```julia
julia> Type{MulticlassLDA} === MulticlassLDA
false

julia> MulticlassLDA |> typeof
UnionAll

julia> Type{MulticlassLDA} |> typeof
DataType
```

Reply:
Ok sorry.